crates/revm/src/db/states/bundle_state.rs

```diff
@@ -151,6 +151,61 @@ impl BundleBuilder {
         self
     }
 
+    /// Set address info of BundleState state.
+    pub fn set_state_address(&mut self, address: Address) {
```

Comment:
since this is a builder type, we can return &mut Self on these functions so they can be chained

the existing `self` functions should then reuse the &mut self ones so we don't duplicate the actual impl

Reply:
Fixed, let me know if this is ok for you